runtime/frame_descriptors.c

```diff
@@ -162,6 +241,26 @@ static caml_frametable_list *cons(
   return li;
 }
 
+/* This function not only create a new caml_frametable_list cell but    */
+/* also make a copy of the new frametable.                              */
+/* Here, we allocate, in a single malloc call, the space for the cons   */
+/* cell and the (appended) frametable copy. This way, we do not have    */
+/* to change the code that unregister the frametable since calling free */
+/* on the cons cell will automatically free the frametable copy at the  */
+/* same time.                                                           */
```

Comment:
Nitpick: the house style is to use block comments for the whole block, not once per line (except in the license header). Could you move to that?